plugins/gatsby-source-nav/gatsby-node.js

```diff
@@ -7,7 +7,7 @@ exports.createSchemaCustomization = ({ actions }) => {
     type Nav {
       id: ID!
       title(locale: String = "en"): String
-      pages: [NavItem!]!
+      pages: [NavItem!]
```

Comment:
It looks like we have a resolver here that ensures this will always return an empty array, even if there aren't child pages: https://github.com/newrelic/docs-website/pull/729/files#diff-c610a67ff8be52857e50c81f69210028602f97bfcf381d404585d458dfb22de4L76

Can you revert this?

Reply:
ah gotcha. seemed like build was failing because of this. going to include release notes link via RootNavigation a la https://github.com/newrelic/docs-website/pull/729#issuecomment-773526770